content/doc/book/managing/system-properties.adoc

```diff
@@ -343,7 +343,7 @@ also enable with extended-read-permission plugin
 See https://jenkins.io/security/advisory/2018-05-09/#SECURITY-786
 
 |hudson.security.LDAPSecurityRealm.groupSearch
-|https://wiki.jenkins.io/display/JENKINS/Features+controlled+by+system+properties#Featurescontrolledbysystemproperties-here[Mouseover]
```

Comment:
The wiki page had a custom tooltip here with the super long value. How do we get it back?

Reply:
Can you provide more details on the contents of that tooltip?

Could we place it in a table cell or some other location to again provide the needed information?

Reply:
It was the default value, a fairly long LDAP query.

Why does the wiki exporter not work for https://wiki.jenkins.io/display/JENKINS/Features+controlled+by+system+properties?

Reply:
> Why does the wiki exporter not work for https://wiki.jenkins.io/display/JENKINS/Features+controlled+by+system+properties?

I don't know.  I wonder if it is related to the redirects that are defined for wiki.jenkins.io?

Reply:
@halkeye any idea why I can't convert https://wiki.jenkins.io/display/JENKINS/Features+controlled+by+system+properties with the wiki exporter but I can convert https://wiki.jenkins.io/display/JENKINS/Git+Client+Plugin ?

Reply:
https://jenkins-wiki-exporter.jenkins.io/confluence-url/https%3A%2F%2Fwiki.jenkins.io%2Fdisplay%2FJENKINS%2FFeatures%2Bcontrolled%2Bby%2Bsystem%2Bproperties.md

No meta page id found

So feels like the redirect to https://www.jenkins.io/doc/book/managing/system-properties/ is happening before wiki exporter gets a chance to read it.

Reply:
```curl -H 'User-Agent: jenkins-wiki-exporter/foo' https://wiki.jenkins.io/display/JENKINS/Features+controlled+by+system+properties
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>301 Moved Permanently</title>
</head><body>
<h1>Moved Permanently</h1>
<p>The document has moved <a href="https://jenkins.io/doc/book/managing/system-properties/">here</a>.</p>
</body></html>```

https://github.com/jenkins-infra/jenkins-infra/blob/staging/dist/profile/templates/confluence/vhost.conf#L2983-L3008

Reply:
Did we really not allow a workaround to access wiki pages, like User-Agent? 😦

Reply:
Its an easy enough fix, just hasn't been done.

Reply:
https://github.com/jenkins-infra/jenkins-infra/pull/1466